region/lowernorfair/east/Main Hall.json

```diff
@@ -132,6 +132,83 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 4],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_navigateHeatRooms",
+        {"or": [
```

Comment:
Can you simplify crossing the room by keeping only the `canTrickyJump` and `SpaceJump` options.

And remove h_navigateHeatRooms please